internal/build/image_builder.go

```diff
@@ -130,8 +130,8 @@ func (d *dockerImageBuilder) applyLabels(df dockerfile.Dockerfile, buildMode doc
 func (d *dockerImageBuilder) addConditionalRuns(df dockerfile.Dockerfile, runs []model.Run, paths []PathMapping) (dockerfile.Dockerfile, []model.Run, error) {
 	consumed := 0
 	for _, run := range runs {
-		if run.Triggers == nil {
-			break
+		if run.Triggers.Empty() {
+			continue
```

Comment:
i'm pretty sure this change is incorrect? This will re-order runs, which is not what you want.

Reply:
can you add a test for this behavior?

Reply:
oh oops, that wasn't supposed to stay in this PR 😅